docs/SRS.md

```diff
@@ -203,17 +203,17 @@ The server shall be available 95% of the time. This also means we have to figure
 
 #### 3.3.2 Defect Rate
 
-Our goal is that we have no loss of any data. This is important so that the users financial data is save, even after a downtime of the server.
+Our goal is that we have no loss of any data. This is important so that the users financial data is safe, even after a downtime of the server.
 
 ### 3.4 Performance
 
 #### 3.4.1 Capacity
 
-The system should be able to manage thousands of requests. Also it should be possible to register as many users as necessary.
+The system should be able to manage thousands of requests. Also, it should be possible to register as many users as necessary.
 
 #### 3.4.2 Storage
 
-Smartphones don't provide much storage. Therefore we are aiming to keep the needed storage as small as possible.
+Smartphones don't provide much storage. Therefore, we are aiming to keep the needed storage as small as possible.
```

Comment:
Maybe we could add somehow add here that this decision affects our database modeling decisions?